apps/services/license-api/src/app/modules/license/dto/verifyLicense.dto.ts

```diff
@@ -12,4 +12,7 @@ export class VerifyLicenseResponse {
   @ApiProperty({ description: 'Is the license verified?' })
   @IsBoolean()
   readonly valid!: boolean
+  @ApiPropertyOptional({ description: 'Optional verifcation data' })
+  @IsOptional()
+  readonly data?: unknown
```

Comment:
Do we want to type this for the cases we know of? I.e. `PassIdentity` or something which is name, national id and image?